src/akamai/js/akamaiHD_flash.js

```diff
@@ -483,8 +483,8 @@ require("../../../html5-common/js/utils/constants.js");
       self.controller.notify(self.controller.EVENTS.SEEKED);
     };
 
-    var raiseBufferingEvent = function() {
-      self.controller.notify(self.controller.EVENTS.BUFFERING);
+    var raiseBufferingEvent = function(event) {
+      self.controller.notify(self.controller.EVENTS.BUFFERING, { url: event.eventObject.url });
```

Comment:
@mitaya, since this was needed for blizzcon, I put in the fix for Akamai for now. Will create a ticket to check and fix it on osmf too.